code/modules/karma/karma_packages.dm

```diff
@@ -36,6 +36,7 @@
 	database_id = KARMAPACKAGE_JOB_BARBER
 	friendly_name = "Barber"
 	karma_cost = 5
+	refundable = TRUE
```

Comment:
And this is why the karma refactor needed to go through
![image](https://user-images.githubusercontent.com/25063394/162329520-749a2e7b-f510-4a0f-9362-2b7a5b9dacb4.png)

So much easier to purge garbage